spec/coordinates_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "spec_helper"
+require_relative "../lib/coordinates"
+
+RSpec.describe Coordinates do
+  describe "valid?" do
+    it "returns false when row is not valid" do
+      input = "d1"
+
+      coordinates = Coordinates.new(input)
+
+      expect(coordinates.valid?).to be false
+    end
+
+    it "returns false when column is not valid" do
+      input = "a4"
+
+      coordinates = Coordinates.new(input)
+
+      expect(coordinates.valid?).to be false
+    end
+
+    it "returns true when row and column are valid" do
```

Comment:
And here, what characteristics make a coordinate valid? I don't think you need to exhaust every possible valid cell, but I think it's worth testing upper and lower case rows.

Reply:
What do you think about this change 4035b4a8b4432bfb9dd86248e85b06289a131d18